libs/core/futures/include/hpx/futures/traits/detail/future_await_traits.hpp

```diff
@@ -26,8 +25,8 @@
 namespace hpx { namespace lcos { namespace detail {
 
     template <typename Promise = void>
-    using coroutine_handle = std::coroutine_handle<Promise>;
-    using suspend_never = std::suspend_never;
+    using coroutine_handle = coro::coroutine_handle<Promise>;
+    using suspend_never = coro::suspend_never;
```

Comment:
Does this file miss a `#include <hpx/config/coroutines_support.hpp>`?

Reply:
It has https://github.com/STEllAR-GROUP/hpx/blob/ce7b0096f7a9354dfb1f6219ec5d2ecd9b8b76da/libs/core/futures/include/hpx/futures/traits/detail/future_await_traits.hpp#L9

Reply:
Yes, I know. We always try to make every file self-contained in terms of the `#include`s it requires. Otherwise removing `#include <hpx/config.hpp>` from the other file (for whatever reason) may break the compilation of this file.